db-migrations/5_log_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS log_game_round (
+	id serial PRIMARY KEY,
+    game_label VARCHAR(255) NOT NULL,
+    player_name VARCHAR(255) NOT NULL,
+    cards_played JSON NOT NULL,
+    cards_purchased JSON NOT NULL
+)
```

Comment:
Try adding a foreign key to reference game_label with the other table: https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-foreign-key/